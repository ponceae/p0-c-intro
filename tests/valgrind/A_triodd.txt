==3257517== Memcheck, a memory error detector
==3257517== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==3257517== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==3257517== Command: ../intro -t 100
==3257517== 
==3257517== 
==3257517== HEAP SUMMARY:
==3257517==     in use at exit: 0 bytes in 0 blocks
==3257517==   total heap usage: 0 allocs, 0 frees, 0 bytes allocated
==3257517== 
==3257517== All heap blocks were freed -- no leaks are possible
==3257517== 
==3257517== For lists of detected and suppressed errors, rerun with: -s
==3257517== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
