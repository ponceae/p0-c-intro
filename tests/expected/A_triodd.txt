odd
2
triodd
4
odd
tri
odd
8
triodd
10
odd
tri
odd
14
triodd
16
odd
tri
odd
20
triodd
22
odd
tri
odd
26
triodd
28
odd
tri
odd
32
triodd
34
odd
tri
odd
38
triodd
40
odd
tri
odd
44
triodd
46
odd
tri
odd
50
triodd
52
odd
tri
odd
56
triodd
58
odd
tri
odd
62
triodd
64
odd
tri
odd
68
triodd
70
odd
tri
odd
74
triodd
76
odd
tri
odd
80
triodd
82
odd
tri
odd
86
triodd
88
odd
tri
odd
92
triodd
94
odd
tri
odd
98
triodd
100
